learning/workflow/export/exporting_for_web.rst

```diff
@@ -89,66 +82,98 @@ Python offers an easy method for this, using ``python -m SimpleHTTPServer``
 with Python 2 or ``python -m http.server`` with Python 3 will serve the current
 working directory on ``http://localhost:8000``.
 
-Locale lookup
-~~~~~~~~~~~~~
+Serving the files
+-----------------
 
-Godot tries to detect the user's locale using information provided by the
-browser, but this is rather unreliable. A better way is to use CGI to read the
-HTTP ``Accept-Language`` header. If you assign its value to the JavaScript
-property ``Module.locale`` after the ``Module`` objects is created, but before
-the engine starts, Godot will use that value to initialize the locale.
-In any case, users should always be offered the option to configure the locale
-manually.
+Exporting for the web generates several files to be served from a web server,
+including a default HTML page for presentation. A custom HTML file can be
+used, see :ref:`doc_customizing_html5_shell`.
+
+The generated ``.html`` file can be used as ``DirectoryIndex`` in Apache
+servers and can be renamed to e.g. ``index.html`` at any time, its name is
+never depended on by default.
+
+The HTML page is designed to fit the game perfectly without cutting off
+parts of the canvas when the browser window is scaled to the game's dimensions.
+This way it can be inserted into an ``<iframe>`` with the game's size, as is
+common on most web game hosting sites.
+
+The other exported files are served as they are, next to the ``.html`` file,
+names unchanged. The ``.wasm`` file is a binary WebAssembly module implementing
+the engine. The ``.pck`` file is the Godot main pack containing your game. The
+``.js`` file contains start-up code and is used by the ``.html`` file to access
+the engine. The ``.png`` file contains the boot splash image. It is not used in
+the default HTML page, but is included for
+:ref:`custom HTML pages <doc_customizing_html5_shell>`.
+
+The ``.pck`` file is binary, usually delivered with the MIME-type
+``application/octet-stream``. The ``.wasm`` file is delivered as
+``application/wasm``.
+
+Delivering the files with server-side compression is recommended especially for
+the ``.pck`` and ``.wasm`` files, which are usually large in size.
+The WebAssembly module compresses particularily well, down to around a quarter
+of its original size with GZip.
```

Comment:
It could be a good idea to mention Brotli as well, which is supported by all modern browsers and can easily be enabled on modern Web servers. It compresses better than gzip overall.